tests/functional/graphql/test_get_slate_lineup.py

```diff
@@ -0,0 +1,84 @@
+from graphql.execution.executors.asyncio import AsyncioExecutor
+from graphene.test import Client
+from fastapi.testclient import TestClient
+
+from app.graphql.graphql import schema
+from app.main import app, load_slate_configs
+from tests.functional.test_dynamodb_base import TestDynamoDBBase
+
+from unittest.mock import patch
+from collections import namedtuple
```

Comment:
Wicked. We're almost ready to merge. 

@gkatre what do you think of this test? Do you think that this reads clearly enough to explain the behavior of the `getSlateLineup` endpoint?